src/app/space-wizard/space-wizard.component.html

```diff
@@ -0,0 +1,191 @@
+                  </div>
+                </div>
+              </div>
+            </form>
+          </div>
+        </div>
+      </div>
+    </section>
+    <footer>
+      <a class="btn btn-default margin-right-5" (click)="cancel()">Cancel</a>
+      <a class="btn btn-default" (click)="wizard.step(wizardSteps.forge.index)">Next</a>
+    </footer>
+  </div>
+  <div class="wizard-step" [class.active]="wizard.activeStep===wizardSteps.forge.index">
+    <header>
+      <h3>Your <strong>BaloonPopGame</strong> Space is ready! How would you like to get started ?</h3>
+    </header>
+    <section class="flex-container in-row-direction flex-grow-1 margin-top-15">
+      <div class="section-column">
+        <header>I'd like to start by Planning my work.</header>
+        <section><span>(more content and graphics)</span></section>
+        <footer><a class="btn btn-default">Planning</a></footer>
+      </div>
+      <div class="section-column">
+        <header>I'd like to start by Coding from scratch.</header>
+        <section><span>(more content and graphics)</span></section>
+        <footer><a class="btn btn-default">New Code</a></footer>
+      </div>
+      <div class="section-column">
+        <header>I'd like to start by Importing existing code.</header>
+        <section><span>(more content and graphics)</span></section>
+        <footer><a class="btn btn-default">Import Code</a></footer>
+      </div>
+      <div class="section-column">
+        <header>I'd like to use a wizard or QuickStart.</header>
+        <section><span>(more content and graphics)</span></section>
+        <footer>
+          <a class="btn btn-default" (click)="wizard.step(wizardSteps.stack.index)">Wizard</a>
+          <a class="margin-top-5 btn btn-default" (click)="wizard.step(wizardSteps.quickStart.index)">QuickStart</a>
+        </footer>
+      </div>
+    </section>
+    <footer>
+      <a class="btn btn-default margin-right-5" (click)="cancel()">Cancel</a>
+      <a class="btn btn-default margin-right-5" (click)="wizard.step(wizardSteps.space.index)">Back</a>
+      <a class="btn btn-default" (click)="finish()">Finish</a>
+    </footer>
+  </div>
+  <div class="wizard-step" [class.active]="wizard.activeStep===wizardSteps.quickStart.index">
+    <header>
+      <h3>Define a QuickStart to add to the <strong>BaloonPopGame</strong> space. </h3>
+    </header>
+    <section class="margin-top-15">
+      <div class="container-fluid">
+        <div class="row">
+          <div class="col-xs-12">
+            <form class="form-horizontal">
+              <div class="form-group has-clear">
+                <label class="control-label col-sm-4">Project type *</label>
+                <div class="col-sm-8">
+                  <select class="form-control" id="projectType" [(ngModel)]="configurator.project.type" name="projectType">
```

Comment:
I think <select> should have the <label> with it, not a level above. If I remember right, in HTML they need to be next to each other to link them. Or you can use "name" attribute, I think.